test/integration/patients.js

```diff
@@ -376,3 +380,37 @@ function HospitalNumber() {
       .catch(helpers.handler);
   });
 }
+
+//testing billing service
+
+function billingServices() {
+  'use strict';
+
+  const patientUuid = '85bf7a85-16d9-4ae5-b5c0-1fec9748d2f9';
+  const billingServiceAttached = 1;
+
+  it('GET /patients/:uuid/services will return a list of the patients billing services', function () {
+    return agent.get(`/patients/${patientUuid}/services`)
+      .then(function (res) {
+        helpers.api.listed(res, billingServiceAttached);
+      })
+      .catch(helpers.handler);
+  });
+}
+
+//testing subsidies
+
+function subsidies() {
+  'use strict';
+
+  const patientUuid = '85bf7a85-16d9-4ae5-b5c0-1fec9748d2f9';
+  const subsidiesAttached = 1;
+
+  it('GET /patients/:uuid/subsidies will return a list of the patients subsidies', function () {
```

Comment:
This is a great example of fixing a bug and updating the tests @IMA-WorldHealth/bhima-core